web/crux/src/domain/agent-installer.ts

```diff
@@ -32,22 +36,41 @@ export class AgentInstaller {
     return `curl -sL ${process.env.CRUX_UI_URL}/api/nodes/${this.nodeId}/script | sh -`
   }
 
-  getScript(name: string): string {
+  getScript(name: string, type: NodeTypeEnum): string {
     this.verify()
 
-    return compileScript({
+    let installScriptParams = {
       name: name.toLowerCase().replace(/\s/g, ''),
       token: this.token,
       insecure: process.env.GRPC_AGENT_INSTALL_SCRIPT_INSECURE === 'true',
-      image: 'ghcr.io/dyrector-io/dyrectorio/agent/dagent:stable',
-    })
+    }
+
+    if (type === 'crane') {
```

Comment:
Use NodeType Enum instead of a hardcoded string.

Reply:
this is a string literal i think :)